src/Tribe/Aggregator/Service.php

```diff
@@ -608,4 +610,57 @@ protected function get_default_origins() {
 
 		return $origins;
 	}
+
+	/**
+	 * Applies a limit to the import request.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args An array of request arguments.
+	 *
+	 * @return mixed
+	 */
+	protected function apply_import_limit( $args ) {
+		if ( isset( $args['limit_type'], $args['limit'] ) ) {
+			return $args;
+		}
+
+		$limit_type = tribe_get_option( 'tribe_aggregator_default_import_limit_type', false );
+		/** @var \Tribe__Events__Aggregator__Settings $settings */
+		$settings = tribe( 'events-aggregator.settings' );
+
+		$limit_args = array();
+		switch ( $limit_type ) {
+			case 'no_limit':
+				break;
+			case 'count':
+				$limit_args['limit_type'] = 'count';
+				$default            = $settings->get_import_limit_count_default();
+				$limit_args['limit']      = tribe_get_option( 'tribe_aggregator_default_import_limit_number', $default );
+				break;
+			default:
+			case 'range':
+				$limit_args['limit_type'] = 'range';
+				$default            = $settings->get_import_range_default();
+				$limit_args['limit']      = tribe_get_option( 'tribe_aggregator_default_import_limit_range', $default );
+				break;
+		}
+
+		/**
+		 * Filters the limit arguments before applying them to the import request arguments.
+		 *
+		 * @since TBD
+		 *
+		 * @param array                              $limit_args The limit arguments.
+		 * @param array                              $args       The import request arguments.
+		 * @param Tribe__Events__Aggregator__Service $service    The service instance handling the import request..
+		 */
+		$limit_args = apply_filters( 'tribe__', $limit_args, $args, $this );
```

Comment:
It looks like this filter might have been accidentally left incomplete? Maybe: `tribe_aggregator_limit_args`